### PR TITLE
Make black run under pre-commit

### PR DESCRIPTION
**Pull request by ssbarnea**
Runs black with **same options** as before under pre-commit.

### CODE REVIEW

**Review by webknjaz — CHANGES_REQUESTED** [review]
I think we need to go through the voting about even having it. The current team never did that.

**Comment by ssbarnea** [comment]
Moved to draft to remove it from daily reminder.

**Review by cidrblock — APPROVED** [review]
I find having all the linters and formatters run the same way, cleaner, and easier to maintain.

This is not an endorsement of black as a formatter in any direction, simply an agreement that this improves the current configuration.

**Review by priyamsahoo — APPROVED** [review]
+1 for this !

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -18,6 +18,17 @@ repos:
   - id: isort
     name: Sort import statements using isort
 
+- repo: https://github.com/psf/black
```
Comment (webknjaz) [inline .pre-commit-config.yaml:21]:
```suggestion
- repo: https://github.com/psf/black.git
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -18,6 +18,17 @@ repos:
...
+  rev: 22.1.0
+  hooks:
+    - id: black
+      language_version: python3
+      args:
+        - -v
+        - --diff
+        - --check
+        - .
```
Comment (webknjaz) [inline .pre-commit-config.yaml:30]:
```suggestion
    - id: black
      language_version: python3
      args:
      - -v
      - --diff
      - --check
      - .
```

**Comment by cidrblock** [comment]
@webknjaz 

"I'll keep hating black separately from this change."

Please submit a PR to have black removed from the repository if you feel strongly about it and seek a quorum approval.

**Pull request merged**